homeassistant/components/bluesound/__init__.py

```diff
@@ -1 +1,76 @@
 """The bluesound component."""
+
+from dataclasses import dataclass
+
+import aiohttp
+from pyblu import Player, SyncStatus
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
+
+
+@dataclass
+class BluesoundData:
+    """Bluesound data class."""
+
+    player: Player
+    sync_status: SyncStatus
+
+
+type BluesoundConfigEntry = ConfigEntry[BluesoundData]
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the Bluesound."""
+    return True
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, config_entry: BluesoundConfigEntry
+) -> bool:
+    """Set up the Bluesound entry."""
+    host = config_entry.data.get(CONF_HOST)
+    port = config_entry.data.get(CONF_PORT)
```

Comment:
They're already there right?

```suggestion
    host = config_entry.data[CONF_HOST]
    port = config_entry.data[CONF_PORT]
```